portindex2json/portindex2json.tcl

```diff
@@ -24,8 +96,13 @@ while {[gets $fd line] >= 0} {
     array set portinfo $line
     array unset json_portinfo
     foreach key [array names portinfo] {
-        if { $key == "categories"} {
-            set json_portinfo($key) [::json::write array {*}$portinfo($key)]
+        if { $key == "categories"  || $key=="variants" || $key=="depends_lib" || $key=="depends_build" || $key=="depends_fetch" } {
```

Comment:
You are using inconsistent whitespacing here. My preference and generally the style used elsewhere is to use whitespace around `==`.

Another way of writing this would be to check whether `$key` is in a list of strings:
```tcl
if {[lsearch -exact {categories variants depends_lib dependes_build dependes_fetch} ${key}] != -1} {
```
even though I'm not sure if that makes the comparison any clearer (in python it would have been nicer, but here the syntax is somewhat strange). You can probably leave it as it is, just fix the spaces.

Well, the "simplification" and maybe better said "generalisation" could have been if you compared `$key` to `depends_*`. There is also `depends_test` and I almost definitely forgot one of the depend types. So: check if `$key` equals categories or variants or `depends_*`.

(I would prefer if one of the more experienced developers commented about that.)

Reply:
Thanks.

Reply:
Perfect compromise.